src/lang/english.txt

```diff
@@ -2346,16 +2348,16 @@ STR_RAIL_TOOLBAR_ELRAIL_CONSTRUCTION_CAPTION                    :Electrified Rai
-STR_RAIL_TOOLBAR_TOOLTIP_BUILD_RAILROAD_BRIDGE                  :{BLACK}Build railway bridge. Shift toggles building/showing cost estimate
-STR_RAIL_TOOLBAR_TOOLTIP_BUILD_RAILROAD_TUNNEL                  :{BLACK}Build railway tunnel. Shift toggles building/showing cost estimate
-STR_RAIL_TOOLBAR_TOOLTIP_TOGGLE_BUILD_REMOVE_FOR                :{BLACK}Toggle build/remove for railway track, signals, waypoints and stations. Hold Ctrl to also remove the rail of waypoints and stations
-STR_RAIL_TOOLBAR_TOOLTIP_CONVERT_RAIL                           :{BLACK}Convert/Upgrade the type of rail. Shift toggles building/showing cost estimate
+STR_RAIL_TOOLBAR_TOOLTIP_BUILD_RAILROAD_TRACK                   :{BLACK}Build Railway Track{NBSP}-{NBSP}{}Places a section of railway in a single direction. Ctrl+Click removes rails. Shift+Click shows estimated cost without purchase
+STR_RAIL_TOOLBAR_TOOLTIP_BUILD_AUTORAIL                         :{BLACK}Autorail Build Mode{NBSP}-{NBSP}{}Allows selecting the direction of rail sections by using the cursor. Ctrl+Click removes rails. Shift+Click shows estimated cost without purchase
+STR_RAIL_TOOLBAR_TOOLTIP_BUILD_TRAIN_DEPOT_FOR_BUILDING         :{BLACK}Build Train Depot{NBSP}-{NBSP}{}For buying and servicing trains. Shift+Click shows estimated cost without purchase
+STR_RAIL_TOOLBAR_TOOLTIP_CONVERT_RAIL_TO_WAYPOINT               :{BLACK}Convert Rail to Waypoint{NBSP}-{NBSP}{}Converts existing section of rail track into a waypoint for redirecting rail vehicles. Ctrl+Click enables merging waypoints. Shift+Click shows estimated cost without purchase
+STR_RAIL_TOOLBAR_TOOLTIP_BUILD_RAILROAD_STATION                 :{BLACK}Build Railway Station{NBSP}-{NBSP}{}For loading and unloading cargo or passengers. Ctrl+Click enables merging stations. Shift+Click shows estimated cost without purchase
+STR_RAIL_TOOLBAR_TOOLTIP_BUILD_RAILROAD_SIGNALS                 :{BLACK}Build Railway Signals -{}Ctrl+Click switches between semaphore/electric variants when a signal type is selected. Click+Drag builds signals along a straight stretch of rail. Ctrl+Click+Drag builds signals up to the next junction or existing signal. Ctrl+Click on an existing signal converts between signal types at no cost. Shift+Click shows estimated cost without purchase
+STR_RAIL_TOOLBAR_TOOLTIP_BUILD_RAILROAD_BRIDGE                  :{BLACK}Build Railway Bridge{NBSP}-{NBSP}{}Places a railway bridge over land or water. Shift+Click shows estimated cost without purchase
+STR_RAIL_TOOLBAR_TOOLTIP_BUILD_RAILROAD_TUNNEL                  :{BLACK}Build Railway Tunnel{NBSP}-{NBSP}{}Underground passage for rail vehicles. Shift+Click shows estimated cost without purchase
```

Comment:
Can maybe detail the operation more here, since it's different from most other build tools: Click on the slope to automatically connect to the other end, no dragging.

Reply:
Are you referring to bridge placement only? If so, agreed!

Reply:
No, tunnel building is the odd one out. (Most of the time changeset comments on GitHub refer to the line just above the comments box.)

Reply:
Better description now!